src/installer.ts

```diff
@@ -188,20 +188,33 @@ async function unzipJavaDownload(
 function getDownloadInfo(
   refs: string[],
   version: string,
+  arch: string,
   javaPackage: string
 ): {version: string; url: string} {
   version = normalizeVersion(version);
+
+  let archExtension = '';
+  if (arch === 'x86') {
+    archExtension = 'i686';
```

Comment:
Where did this extension come from? Is it not `x86`?

Reply:
`i686` is the latest iteration of the `IA-32` (commonly known as `x86`). From today's perspective, it's a synonym of `x86` (although it is technically not). I didn't choose this extension, I just looked at https://cdn.azul.com/zulu/bin/ and saw that's what they use.